docs/content/contributing/tests.md

```diff
@@ -0,0 +1,89 @@
+
+!!! info "WIP: Test Suite Refactoring"
+
+    We are currently in the process of parallelizing all of our tests. Tests will be moved into `test/tests/parallel/` and new tests should be placed there as well.
+
+### How Are Tests Run?
+
+Before running tests yourself, you need to understand the following first: we need to differentiate between
+
+1. running test locally on **your system**
+2. running tests in **our CI** (_GitHub Actions_), e.g. when [testing a pull request][testing-prs]
+
+#### How Local Tests Work
+
+Local tests are executed on your machine. In the ["Prerequisites" section](#prerequisites) below you'll find everything you need to run tests yourself. Tests under `test/tests/parallel/` are run in parallel. To be more precise, tests within a single file are run **sequentially**, but multiple files are run in **parallel**. Parallel tests are partionioned into sets (currently 3). You can run all sets simultaneously, although we do not recommend this because of the high resource demand this incours on your system, which in turn could lead to test failures. The tests under `test/tests/serial/` are all run in serial.
+
+#### How Tests Work With GitHub Actions
+
+With GitHub Actions, is is very similar to how [tests run locally](#how-local-tests-work), but all parallel sets and the serial tests are run **in parallell** but separately (on different runners) to not interfere with each other. This increases the degree of parallelization which in turn speeds up CI.
+
+## Running Tests
+
+### Prerequisites
+
+To run the test suite, you will need to
+
+1. [Install Docker][get-docker]
+2. Install `jq` and (GNU) `parallel` (under Ubuntu, use `sudo apt-get -y install jq parallel`)
+3. Execute `git submodule update --init --recursive` if you haven't already initialized the git submodules
+
+### Executing Test(s)
+
+We use `make` to run commands. You will first need to build the container image via `make build`. You can then
+
+1. Run all tests: `make clean tests`
+2. Run a single test: `make clean generate-accounts test/<TEST NAME WITHOUT .bats SUFFIX>`
+3. Run multiple unrelated tests: `make clean generate-accounts test/<TEST NAME WITHOUT .bats SUFFIX>,<TEST NAME WITHOUT .bats SUFFIX>` (just add a `,` and then immediately write the new test name)
+4. Run a whole set or all serial tests: `make clean generate-accounts tests/parallel/setX` where `X` is the number of the set or `make clean generate-accounts tests/serial`
+
+??? example "Run One or Multiple Specific Test(s)"
+
+    Multiple test files can be run sequentially with a `,` delimiter between file names: `make test/tls_letsencrypt,tls_manual`
+
+    To run only the tests in `template.bats`, use `make clean test/template` (or with relative path: `make clean test/parallel/set2/template`).
```

Comment:
Isn't this redundant compared to the listing directly above?

Reply:
Yes, yes it is. I will remove it.

Reply:
Done in 741c116